tabby-settings/src/components/multiHotkeyInput.component.ts

```diff
@@ -1,6 +1,7 @@
 import { Component, Input, Output, EventEmitter, ChangeDetectionStrategy } from '@angular/core'
 import { NgbModal } from '@ng-bootstrap/ng-bootstrap'
 import { HotkeyInputModalComponent } from './hotkeyInputModal.component'
+import { Hotkey } from 'tabby-core/src/api/hotkeyProvider'
```

Comment:
`Hotkey` should be barrel-exported through `tabby-core/src/index.ts`, same as everything else